articles/azure-functions/durable-functions-disaster-recovery-geo-distribution.md

```diff
@@ -0,0 +1,81 @@
+- This scenario covers outages at the compute layer, but the storage account continues to be the single point of failure for the Function App. If there is a Storage outage, the application suffers a downtime.
+- If the FunctionApp is failed over, there will be increased latency since it will access its storage account across regions.
+- Accessing the storage service from a different region where it is located incurs in higher cost due to network egress traffic.
+- This scenario depends on Traffic Manager. Considering [how Traffic Manager works](../traffic-manager/traffic-manager-overview#how-traffic-manager-works), it may be some time until a client application that consumes a Durable Function needs to re-query the FunctionApp address from Traffic Manager. 
+
+
+## Scenario 2 - Load Balanced Compute with Regional Storage
+The previous proposed scenario covers only the case of failure in the compute services. If the storage service fails, it will result in an outage of the FunctionApp.
+To ensure continuous operation of the durable functions, this scenario uses a local storage account on each region to which the FunctionApps are deployed.
+
+![Diagram showing scenario 2.](media/durable-functions-disaster-recovery-geo-distribution/durable-functions-geo-scenario02.png)
+
+This approach adds improvements on the previous scenario:
+- In the case of failure, failover to the failover regions is still managed by Traffic Manager. However, because the FunctionApp relies on its own storage account, the durable functions continue to work.
+- During a failover, there is no additional latency in the failover region, since the FunctionApp and the storage account are co-located.
+- Failure of the storage layer will cause failures in the durable functions, which, in turn, will trigger a redirection to the failover region. Again, since the FunctionApp and storage are isolated per region, the durable functions will continue to work.
+ 
+Important considerations for this scenario:
+- If the Function App is deployed using a dedicated AppService plan, there is an increased cost by replicating the compute instance in the failover datacenter.
+- Current state is not failed over. This implies that executing and checkpointed functions will fail. It is up to the client application to retry/restart the work.
+
+## Scenario 3 - Load Balanced Compute with GRS Shared Storage
+This scenario is a modification over the first scenario, implementing a shared storage account. The main difference that the storage account is created with geo-replication enabled.
+Functionally, this scenario provides the same advantages as Scenario 1, but it enables additional data recovery advantages:
+- Geo-redundant storage (GRS) and Read-access GRS (RA-GRS) maximize availability for your storage account. Additionally, RA-GRS provides read-only access to the data in the secondary location, in addition to geo-replication across two regions.
```

Comment:
The sentence beginning with Additionally is confusing.  Geo-location across two regions implies you have access to the storage in the two regions, so why is read-only access "additional"?

Reply:
check (removed the comment)